### DISCUSSION TITLE
Why page router does not natively support antd references but app router works?

### DISCUSSION THREAD

**Discussion by Autumn00air** [vercel/next.js#62189] [App Router]
I am using the next application created by the latest next.js. Whether I use page router or application router, it is successful when it is started, there is no doubt about it. However, when I reference the antd library, the application router still work, but the page router reports an error:  SyntaxError: Cannot use import statement outside a module. It is 2024 now!!!

**Comment by mutasim77** [comment]
check it out: [link](https://ant.design/docs/react/use-with-next#using-pages-router) It should work. If not, please share your code exactly where the error is occurring.

**Reply by Autumn00air** [reply]
I tried it on this basis, but the page router still failed while the application router make it. Now I considered that it might be because of the version problem of antd or next.js, so I tried to roll back their version respectively. Finally, I found that when I downgraded next.js from v14 to v13, it did not report this error. It works. that's all○|￣|_
